src/main/java/com.company/tree/TreeNode.java

```diff
@@ -0,0 +1,17 @@
+package com.company.tree;
+
+import lombok.AllArgsConstructor;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+public class TreeNode {
+    int val;
```

Comment:
Keep the data members as public, if you are doing direct access to it.

Reply:
changed data members as public